modules/doc/content/getting_started/installation/install_libtorch.md

```diff
@@ -18,20 +22,29 @@ ldd --version
 
 ## Setting up the environment
 
-For Mac workstations, the user needs to create a conda environment using the
-instructions [here](installation/conda.md). On Linux machines, however,
-we cannot use the conda packages due to the mismatch between `libc` versions.
-For this reason, given that the system `libc` version allows the linking between
-the two libraries, we need to install `PETSc` and `libMesh` manually. For instructions
-on the installation of these, see [installation/hpc_install_moose.md].
-
-In situations when the `libc` version allows the linking but the compiler stack
-has been compiled with an older `libc` version (HPC machines potentially), we need to build the
-compiler from scratch. For instructions in this process, visit [installation/manual_installation_gcc.md]
+For both Mac and Linux workstations, the user needs to create a conda environment using the
+instructions [here](installation/conda.md).
```

Comment:
Sorry to nitpick, but over the years of content creation I have learned it is bad practice to use "click [here]()" type links.

I would suggest naming the anchor the actual title of the target page:

For both Mac and Linux workstations, the user needs to follow the [Conda MOOSE Environment](installation/conda.md) instructions.